releasing/plugin-commands-publishing/src/pack.ts

```diff
@@ -127,6 +127,21 @@ async function readReadmeFile (filesMap: Record<string, string>) {
   return readmeFile
 }
 
+function getGzipConfig () {
+  const gzipConfig: ZlibOptions = {}
+
+  const level = process.env.PNPM_PACK_GZIP_LEVEL
+  if (level !== undefined) {
+    const parsedLevel = parseInt(level)
+    if (isNaN(parsedLevel)) {
+      throw new PnpmError('INVALID_GZIP_LEVEL', `Invalid gzip level: '${level}'`)
```

Comment:
Let me know if I need to register this error somewhere,

A quick full-text search didn't yield exactly one instance of other errors.